src/PlayersManager.ts

```diff
@@ -712,17 +712,16 @@ export class PlayersManager extends Dispatcher {
         const captionsTracks = tracks.find(
           (track) => track._kind === "subtitles"
         );
-        if (
-          captionsTracks ||
-          (audioTracks &&
-            audioTracks._language &&
-            audioTracks._language != undefined)
-        ) {
-          // this video has captions or audio
+        if (audioTracks && audioTracks._language && audioTracks._language != undefined) {
```

Comment:
audioTracks._language will fail anyway if audioTracks._language is undefined. so i think you can just shorten to 
audioTracks && audioTracks._language !== undefined

Reply:
same as the previous comment, i don't want to change old functionality, this is the way it worked before i just move it to another condition. if you still want i can change it.

Reply:
it's not changing functionality. the 3rd condition would never be called if language is undefined. you can test it yourself.

Reply:
you are right, done.